testsuite/features/build_validation/add_common_repositories/alma9_minion_add_common_repositories.feature

```diff
@@ -2,7 +2,7 @@
 # Licensed under the terms of the MIT license.
 
 @alma9_minion
-Feature: Add the Alma 9 distribution custom repositories
+Feature: Add the Alma 9 distribution common repositories
```

Comment:
Those are nor common, but custom channels.
Common channels are those created with spacewalk-common-channels command

Reply:
OK, so this is basically a terminology problem.

I don't really care about the terminology (on the contrary, I would love to see it clarified).

My main concern and the reason I asked this refactoring is to have a bit of order and remove the clutter we have in `testsuite/features/build_validation/add_custom_repositories`, where we mix features from two different pipeline stages: 
```
                stage('Add MUs')
                stage('Add Common Channels')
```
matching those two build parameters:
```
must_add_custom_channels
must_add_common_channels
```
As you can see @srbarrios the name used here is "common channels". If that's improper, fair enough, but then let's fix it everywhere.

Reply:
I agree that we need to rename the files to follow the naming conventions.
But it could perfectly go in a different PR, as it solve a different issue.

"Add MUs"
Should contain the creation of custom repositories plus custom channels using as source of information the JSON content with the Maintenance Updates.
Nothing else.

"Add common channels":
Should contain the execution of spacewalk-common-channel command for those channels that we have officially documented to be reposynched that way, instead of using our usual WebUI Product Wizard

"Build CLM projects":
Must be a new stage after "Core" stage, that includes all the CLM projects necessary to be used later on during the creation of AKs for some products, like RH-like products containing Modules repositories that we want to filer.